aiortc/rtcrtptransceiver.py

```diff
@@ -21,6 +43,7 @@ def __init__(self, kind, receiver, sender, direction='sendrecv'):
         self.__receiver = receiver
         self.__sender = sender
         self.__stopped = False
+        self.__stream = MediaStream()
```

Comment:
how does this help? I don't see what we gained as the `stream` attribute is immutable?